Slam/Queue/APIManagedQueue.swift

```diff
@@ -9,10 +9,23 @@ public class APIManagedQueue: QueueManager {
         self.queueView = queueView
     }
 
+    public init() {
```

Comment:
We would probably need to shift to using the other initializer in order to make `queueView` not an Optional.